src/test/java/com/jcabi/aspects/ImmutableTest.java

```diff
@@ -89,6 +91,35 @@ public void catchesTypesMutableByClassInheritance() {
     }
 
     /**
+     * Test Version.
+     */
+    @Test
+    public void testVersion() {
+        try {
+            new Mutable();
+        } catch (final IllegalStateException ex) {
+            MatcherAssert.assertThat(
+                ex.getMessage(),
+                Matchers.not(
+                    Matchers.containsString(
+                        "${projectVersion}"
+                    )
+                )
+            );
+            MatcherAssert.assertThat(
+                ex.getMessage(),
+                Matchers.not(
+                    Matchers.containsString(
+                        "${buildNumber}"
+                    )
+                )
+            );
+            return;
+        }
+        MatcherAssert.assertThat("should not happen.", false);
```

Comment:
@pecko @pinaf This is bad style, to use `catch` for expecting an exception and asserting on it. Instead you should be using the JUnit Rule `ExpectedException`.